### DISCUSSION TITLE
Updating local installation from a web location

### DISCUSSION THREAD

**Discussion by shlomgad** [nroduit/Weasis#450] [Q&A]
Hi,
Up until version 3.8.1 there was an ability to update the local native launcher when launched from a web context (the weasis protocol) by providing a 'cdb' config parameter and pointing it to a url that holds the contents of weasis.war.
Starting with version 4.0.0 the weasis.war archive is no longer there and I was wandering if there is a way to point the native launcher to a web resource for updates or for retrieval of customized resources.
Thanks

**Comment by nroduit** [comment]
Yes, it's still possible, except that the weasis.war package is no longer distributed, but [can be built](https://weasis.org/en/getting-started/building-weasis/).

However, we will be proposing a new weasis-manager component by the end of the year. This component is the successor to weasis-pacs-connector and will manage and user and launch preferences by user, group or host. It will also manage minor version updates according to [this compatibility file](https://github.com/nroduit/Weasis/blob/master/weasis-distributions/etc/config/version-compatibility.json) and by importing weasis-native.zip (from weasis 3.6.0). In other words, even if the computer park has heterogeneous Weasis versions, it will be able to update the different versions according to the compatibility file. This is useful when updating with the native version installer on workstations is done progressively.
